integration-tests/features/Organisations.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Features related to Organisation Management
+
+  @createOrganisations
+  Scenario: POST to /organisations with valid body returns created org
+    Given I generate a json payload called 'newOrgRequest'
+    When I make a POST to the function at '/organisations'
+    Then I should get a status code 201
+    And the response should be a superset of all the keys and values set from 'newOrgRequest'
+
+  @createOrganisations @errorHandling
+  Scenario: POST to /organisations with invalid body returns bad request
+    Given I generate a json payload called 'badNewOrgRequest'
+    When I make a POST to the function at '/organisations'
+    Then I should get a status code 400
+
+  @getOrganisationsById
+  Scenario: GET to /organisations with pre-conditioned knownOrgId returns a known org
+    When I make a GET to the function at '/organisations/knownOrgId'
+    Then I should get a status code 200
+
+  @getOrganisationsById @errorHandling
```

Comment:
I presume this includes the scenario when no organisation Id is passed in?

Reply:
Because these are expensive (in terms of time and resources) tests, this only tests a handful of scenarios. In this case, we don't test for a null organisation here but we should do it alot lower down in the pyramid. I hadn't relaised I forgot to write that unit test, but I've added it now: https://github.com/math-dojo/user-account-service/blob/39ffd4dd68211600fa26405d69a7b430d88614e6/src/test/java/io/mathdojo/useraccountservice/services/OrganisationServiceTest.java#L75